fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidHavingBuilder.java

```diff
@@ -28,8 +29,7 @@
     /**
      * Build a having model that ANDs together having queries for each of the metrics.
      *
-     * @param metricMap  A map of logical metric to the set of havings for that metric
-     *
+     * @param metricMap A map of logical metric to the set of havings for that metric
```

Comment:
two spaces between @param names and comments is correct to our style guide.  Your formatter reduced to a single space incorrectly.